src/Adapter/Attribute/AttributeDataProvider.php

```diff
@@ -84,6 +84,8 @@ public static function getAttributeIdsByGroup($id_group, $not_null = false)
      */
     public function getProductCombinations($idProduct)
     {
+        $locales = $this->container->get('prestashop.adapter.legacy.context')->getLanguages();
```

Comment:
Yep I see… I don't like the old "everyone do that", but its not the more important modification to do right now.